File: Module3/3A(JavaScript)/JSCodingChallenge/challenge2.js
// Create a length converter function.

function lengthConverter(meter){
    // Convert meters to centimeters (1 meter = 100 centimeters)
    const centimeter = meter * 100;
    return centimeter;
}

const length = 41;
const convertedLength = lengthConverter(length);

console.log(length + " meters is equivalent to " + convertedLength + " centimeters");

File: Module4/Tasks_4C/PythonCodingChallenge/challenge2.py
def length_converter(meter):
    # Convert meters to centimeters (1 meter = 100 centimeters)
    centimeter = meter * 100
    return centimeter

length = 27
converted_length = length_converter(length)

print(f"{length} meters is equivalent to {converted_length} centimeters")
